screens/CategoryScreens/CreateCategoryScreen.tsx

```diff
@@ -36,85 +33,72 @@ export default function CreateCategoryScreen({ navigation }: RootStackScreenProp
     })
   })
 
-  const {data: categoriesData, refetch} = useQuery<GetCategoriesQuery>(GetCategoriesDocument, {
+  const { data: categoriesData } = useQuery<GetCategoriesQuery>(GetCategoriesDocument, {
     variables: { passwordHash: passwordHash },
     onError: (error => {
       Alert.alert(error.message);
     })
   });
 
-  useEffect(() => {
-    let newColor = selectedColor.substring(1);
-    setColor(newColor);
-  }, [selectedColor]);
+  const categoryTaken = () => {
+    if (categoriesData?.categories.__typename === 'CategoriesSuccess') {
+      return true && categoriesData.categories.categories.find((cat) => {
```

Comment:
you dont need to return 'true &&', it will always just be equivalent to the second argument